.github/workflows/ci.yaml

```diff
@@ -25,11 +25,13 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v2
         with:
-          go-version: "1.14.6"
+          go-version: "1.17"
       - name: Generate
         run: make generate
+      - name: go clean -testcache
+        run: go clean -testcache
       - name: go test
-        run: go test -coverprofile=coverage.txt ./...
+        run: go test -coverprofile=coverage.txt ./... -v
```

Comment:
can you drop this one? I'm not sure we really need it. I don't really care to see the test names when there's no failure (I don't even look at the test run if there's no failure tbh). Go does the right thing when there's failure and only expands the failing tests. And actually having `-v` in there just adds a ton of noise when there is a test, printing out the names of all the passing tests.

Reply:
sure, It was in the Makefile, so I added for consistency

Reply:
ok makes sense. consistency > sense :D